microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -542,7 +542,7 @@ def _install(self) -> bool:
         node.execute(
             "ninja",
             cwd=self.dpdk_build_path,
-            timeout=1200,
+            timeout=1800,
```

Comment:
Can you share the logs to the tests that are failing after 20? 30 minutes seems very long

Reply:
Edit, 30 minutes is needed for the build in some cases